packages/zostso/src/TsoResponseService.ts

```diff
@@ -65,12 +65,12 @@ export class TsoResponseService {
             messages: isNullOrUndefined(collectedResponses)? "" : collectedResponses.messages
         };
 
-        if (isNullOrUndefined(zosmfResponse.servletKey) && zosmfResponse.msgData) {
+        if (!isNullOrUndefined(zosmfResponse.servletKey)) {
```

Comment:
Since we are already changing things around a bit, would it be appropriate to replace the deprecated isNullOrUndefined here?